src/vm/luca_loaderwriter.cpp

```diff
@@ -0,0 +1,75 @@
+#include "common.h"
+#include "luca_loaderwriter.h"
+#include "pal.h"
+
+LoaderWriter::LoaderWriter(int pid) {
+    LIMITED_METHOD_CONTRACT;
+
+    this->pid = pid;
+
+    SString path;
+    path.Printf("loader-%d.map", pid);
```

Comment:
You should specify the full path to the file.  In this case, the file should go in /tmp, and you can get the temp file path via a call to GetTempPath.  See PerfMap::PerfMap for an example.